lessons/modelevaluation.ipynb

```diff
@@ -0,0 +1,786 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "# Evaluating Models\n",
+    "\n",
+    "Goals:\n",
+    "\n",
+    "* Be able to design and carry out tests of model adequacy (goodness of fit), and comparisons between models\n",
+    "\n",
+    "* Understand and be prepared to use the Bayesian Evidence"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "## Further reading\n",
+    "\n",
+    "* Mackay, Chapters 3 and 28\n",
+    "\n",
+    "* Gelman et al (BDA3), Chapter 6"
```

Comment:
BDA should go first, as our preferred-though-not-required text.

Reply:
Agreed